containers/allan_dip_bot/run_bot.py

```diff
@@ -274,7 +281,18 @@ def main() -> None:
     parser.add_argument(
         "--outdir", type=Path, help="output directory for game json to be stored"
     )
+    parser.add_argument(
+        "--outdir", type=str, help="output directory for game json to be stored"
+    )
```

Comment:
This is a duplicate definition and should be removed. It's likely a mistake from the merge.

Reply:
done